presto-hive-hadoop2/bin/test.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
If this is supposed to be run by developers (not only on Travis), this should be 
```
#!/usr/bin/env bash
```

otherwise eg on my Mac this will use bash 3.x instead of 4.x and `[[ -v` might not work

(this of course applies to all the scripts)